packages/api/tests/endpoints/test-rules.js

```diff
@@ -290,8 +276,10 @@ test('POST creates a rule', async (t) => {
   t.deepEqual(record, newRule);
 });
 
-test('POST returns a record exists when one exists', async (t) => {
-  const newRule = { ...testRule };
+test('POST returns a 409 response if record already exists', async (t) => {
+  const newRule = fakeRuleFactoryV2();
```

Comment:
My goal with some of these changes to make these tests a little less inter-dependent